holoviews/core/spaces.py

```diff
@@ -413,8 +416,22 @@ class Callable(param.Parameterized):
     inputs = param.List(default=[], doc="""
          The list of inputs the callable function is wrapping.""")
 
+    def __init__(self, **params):
+        super(Callable, self).__init__(**params)
+        self._memoized = {}
+
     def __call__(self, *args, **kwargs):
-        return self.callable_function(*args, **kwargs)
+        inputs = [inp for inp in self.inputs if isinstance(inp, DynamicMap)]
+        streams = [s for inp in inputs for s in get_nested_streams(inp)]
+        values = tuple(tuple(sorted(s.contents.items())) for s in streams)
```

Comment:
It really doesn't matter much, it just needs to be consistently ordered. If we're using it somewhere else it can use the same scheme or another scheme without having any effect here.